src/processor/result/factorized_table.cpp

```diff
@@ -245,29 +245,41 @@ void FactorizedTable::updateFlatCell(
     }
 }
 
+void FactorizedTable::copySingleValueToVector(ft_tuple_idx_t tupleIdx, ft_col_idx_t colIdx,
```

Comment:
Actually, i can't use a memcpy to copy a value to a vector directly since it doesn't set the overflow ptr correctly.
Instead we should use another utility function: `ValueVectorUtils::copyNonNullDataWithSameTypeIntoPos` to do this.